google/cloud/internal/format_time_point.cc

```diff
@@ -26,23 +27,37 @@ std::string FormatFractional(std::chrono::nanoseconds ns) {
     return "";
   }
 
-  char buffer[16];
+  using std::chrono::milliseconds;
+  using std::chrono::nanoseconds;
+  using std::chrono::seconds;
+  constexpr int kMaxNanosecondsDigits = 9;
+  auto constexpr kBufferSize = 16;
```

Comment:
Nit: Why `auto` on this one and `int` on the others?

Reply:
Fixed, here and elsewhere. Thanks!